templates/common/_base/units/ovs-vswitchd.service.yaml

```diff
@@ -0,0 +1,8 @@
+name: ovs-vswitchd.service
+dropins:
+  - name: 10-ovs-vswitchd-restart.conf
+    contents: |
+      [Service]
+      {{- if or (eq .NetworkType "OVNKubernetes") (eq .NetworkType "OpenshiftSDN") }}
```

Comment:
I'm not sure why we need conditionals on the dropins, the services are enabled under the very same conditions, so if network isn't sdn|ovn, the service doesn't even get enabled and doesn't matter if it has restart always or not

Reply:
ack